[docs] Gh #5196 describe current limitations (#5428)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/selector/withtext.md

// and then the child div (.child)
Selector('div').withText('foo');
```

> If you want to target elements with HTML symbols or HTML entities (like `&nbsp;`, newline chars), refer to the [Select Elements That Contain Special Characters](../../../guides/basic-guides/select-page-elements.md#select-elements-that-contain-special-characters) section.

Reviewer: ```suggestion
To target elements with HTML symbols or HTML entities (e.g., `&nbsp;`, newline chars), refer to the following section: [Select Elements That Contain Special Characters](../../../guides/basic-guides/select-page-elements.md#select-elements-that-contain-special-characters).
```

// and then the child div (.child)
Selector('div').withText('foo');
```

> If you want to target elements, containing special HTML characters, refer to the [Select Elements That Contain Special Characters

Author: done

> If you want to target elements, containing special HTML characters, refer to the [Select Elements That Contain Special Characters

Author: Edited

> ](../../../guides/basic-guides/select-page-elements.md#select-elements-that-contain-special-characters) section.

Reviewer 2: Accidental new line

Author: fixed, thanks!
